docs/reference/alert-schema.asciidoc

```diff
@@ -112,7 +112,12 @@ Type: keyword
 |`signal.rule.type` |`kibana.alert.rule.type`| Type: keyword
 |`signal.rule.updated_at` |`kibana.alert.rule.updated_at`| Type: date
 |`signal.rule.updated_by` |`kibana.alert.rule.updated_by`| Type: keyword
-|`signal.rule.version` |`kibana.alert.rule.version`| Type: keyword
+|`signal.rule.version` |`kibana.alert.rule.version`| A number corresponding to the version of a rule distributed by Elastic.
+
+Type: keyword
+|N/A |`kibana.alert.rule.revision` | A number that gets incremented each time you edit a rule.
```

Comment:
Does this apply to custom and prebuilt rules? If it's both, that might be worth clarifying, for example:

```suggestion
|N/A |`kibana.alert.rule.revision` | A number that gets incremented each time you edit a custom or prebuilt rule.
```

Reply:
I think so. @spong can you confirm?